alcs-api/src/application/application-decision/application-decision.entity.ts

```diff
@@ -18,6 +18,18 @@ export class ApplicationDecision extends Base {
   @Column({ type: 'timestamptz' })
   date: Date;
 
+  @AutoMap()
+  @Column({ type: 'timestamptz', nullable: true })
+  auditDate: Date;
```

Comment:
This can be misleading, currently, all fields prefixed with 'audit' are for technical auditing

Reply:
The label for it is Audit Date in the ui, any suggestions for what to call it?